### PR TITLE
Add help options and hubspot integration

### PR DESCRIPTION
**Pull request by jyash97**
#64

### CODE REVIEW

File: src/pages/SandboxPage/SandboxPage.js [hunk 1]
```diff
@@ -47,7 +47,25 @@ class SandboxPage extends Component {
 		}
 
 		return (
-			<SearchSandbox appId={appId} appName={appName} credentials={credentials} isDashboard>
+			<SearchSandbox
+				appId={appId}
+				appName={appName}
+				credentials={credentials}
+				isDashboard
+				customProps={{
```
Comment (metagrover) [inline src/pages/SandboxPage/SandboxPage.js]: Why do we have `customProps` here?

File: src/pages/HelpChat/HelpChat.js [hunk 2]
```diff
@@ -0,0 +1,177 @@
+import React from 'react';
```
Comment (metagrover) [inline src/pages/HelpChat/HelpChat.js]: Is `HelpChat` a page or a component? 🤔

**Comment by siddharthlatest** [comment]
@jyash97 This PR has conflicts, can you resolve them and update the PR?

**Comment by jyash97** [comment]
#69 Check this one

**Pull request closed without merging**